TurkishFilter/sections/css_extended.txt

```diff
@@ -39,6 +39,7 @@ hdfilmcehennemi.com#?##sidebar > .textwidget2:has(> a[href][target="_blank"] > i
 duzcetv.com#?#body > div.container:has(> ins.adsbygoogle)
 marastahaber.com#?#div[style="float: right;"] > a[href][target="_blank"] > video:has(> source[src^="http://marastahaber.com/wp-content/uploads/"][src$="/MaviTekReklam.webm"])
 dizibox.pw#$?##vido-before-video { remove: true; }
+fenokulu.net#?#.banner300Soru:upward(div)
```

Comment:
add also the rule for platforms without ExtCSS

Reply:
Mobile layout has no such elements, is it necessary?

Reply:
`upward` works on latest Windows `.exe` / macOS / `.apk` (full) app?